docs/how-to/configure.md

```diff
@@ -230,6 +230,14 @@ To connect your PMM server to Percona Platform:
     ![!image](../_images/PMM_Settings_Percona_Platform_Login.png)
 2. Fill in the *Connect PMM to Percona Portal* form with the name of your PMM instance and the credentials of your Percona Account:
 3. Click *Connect*.
+To confirm that you have successfully connected the server and check the list of all servers currently connected to an organization, go to [Percona Platform](https://portal.percona.com) > **Dashboard** tab and click **View Instances** next to the **Connect your PMM** step. 
+
+### Disconnect a PMM instance
+ Disconnect a PMM instance when you want to unlink it from your Percona Platform organization or stop monitoring it there. 
```

Comment:
this file showed up because I branched from "main" instead of  2.26.0. This file should not have been picked up so it now shows up as outdated and can be ignored.